test/es6/exportmodule.js

```diff
@@ -0,0 +1,2 @@
+export var x;

```

Comment:
Jenkins *shouldn't* like it. Apparently it didn't see an issue... I'm looking into it.